src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -35,24 +50,38 @@ public Book create(@RequestBody Book book) {
     }
 
     @GetMapping("/{id}")
-    public Book findOne(@PathVariable Long id) {
-        return bookRepository.findById(id).orElseThrow(BootstrapMethodError::new);
+    public Book findOne(@PathVariable Long id) throws IOException {
+        return bookRepository.findById(id).orElseThrow(this::bookNotFound);
```

Comment:
Maybe this exception can be handled with a custom one with a more information (more readable) not throwing it at the framework level